CMakeLists.txt

```
@@ -70,10 +70,16 @@ endif()
 # https://cmake.org/cmake/help/v3.12/module/CPack.html
 # https://cmake.org/cmake/help/v3.7/module/CPackRPM.html
 include(GNUInstallDirs)
-set(CMAKE_INSTALL_PREFIX "/usr")
+
+set(SHARED_CONFIG_PATH "${CMAKE_INSTALL_FULL_DATAROOTDIR}/touchegg/touchegg.conf")
```

Could you rename it to `SYSTEM_CONFIG_FILE_PATH` please? Just to be consistent with the name used in code:

```suggestion
set(SYSTEM_CONFIG_FILE_PATH "${CMAKE_INSTALL_FULL_DATAROOTDIR}/touchegg/touchegg.conf")
```